backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interest/InterestComputationService.java

```diff
@@ -321,7 +322,7 @@ private InvoiceAllocations initInvoiceAllocations(
 		Check.assumeNotNull(invoice.getAmount(), "Invoices with no amount should've been skipped already! LogId="
 				+ invoice.getId());
 
-		final Instant conversionDate = Instant.now();
+		final Instant conversionDate = SystemTime.asInstant();
```

Comment:
SystemTime.asInstant() is way better than Instant.now().... but pls try avoiding it as much as possible.

What stops you from getting the "computingDate" from request/param?